src/Umbraco.Web.UI.Client/src/views/users/user.controller.js

```diff
@@ -374,7 +374,7 @@
         }
 
         function enableUser() {
-            vm.enableUserButtonState = "busy";
+            vm.enableUserButtonState = "busfy";
```

Comment:
@nielslyngsoe shouldn't this be `busy` as in commit message?
https://github.com/umbraco/Umbraco-CMS/blob/v9/contrib/src/Umbraco.Web.UI.Client/src/views/users/user.controller.js#L377

Here it is `busy`:
https://github.com/umbraco/Umbraco-CMS/blob/v9/contrib/src/Umbraco.Web.UI.Client/src/views/users/views/users/users.controller.js#L339